db/migrate/20220707204323_change_foreign_key_for_posts.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeForeignKeyForPosts < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :posts, :user_id , :author_id
+  end
+end
```

Comment:
### Kindly Note that You can solve the above issues with your own solution but make sure this error below is not present and Your tables match what the requirements say.

![image](https://user-images.githubusercontent.com/65068771/177881566-824b98b6-de5e-4d57-8356-67668c7160de.png)